### PR TITLE
readme: update links in readme

### PR DESCRIPTION
**Pull request by dcalvin**
## What have you changed? (mandatory)

Please explain **IN DETAIL** what the changes are in this PR and why they are needed:
- This PR updated the links in readme.  Also added a missing file.  Related PR: https://github.com/tikv/tikv/pull/4929

## What are the type of the changes? (mandatory)
-  Doc update

## Does this PR affect documentation (docs) or release note? (mandatory)

- Docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,22 +71,22 @@ We provide multiple deployment methods, but it is recommended to use our Ansible
...
-    - [Deploy TiKV Using Binary Files](docs/op-guide/deploy-tikv-using-binary.md): To deploy a TiKV cluster using binary files on a single node or on multiple nodes
+    - [Deploy TiKV Using Docker Compose](docs/how-to/deploy/using-docker-compose.md): To quickly test TiKV separately without TiDB using [`tidb-docker-compose`](https://github.com/pingcap/tidb-docker-compose/) on a single machine
+    - [Deploy TiKV Using Docker](docs/how-to/deploy/using-docker.md): To deploy a multi-node TiKV testing cluster using Docker
+    - [Deploy TiKV Using Binary Files](docs/how-to/deploy/using-binary.md): To deploy a TiKV cluster using binary files on a single node or on multiple nodes
 
 ### Production deployment
 
 For the production environment, use [Ansible](https://github.com/pingcap/tidb-ansible) to deploy the cluster.
 
 - [Deploy TiDB Using Ansible](https://github.com/pingcap/docs/blob/master/op-guide/ansible-deployment.md)
```
Comment (overvenus) [inline README.md]: Is it a dead link?

File: docs/how-to/deploy/using-docker-compose.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    depends_on:
+      - "pd0"
+    restart: on-failure
+```
+
+All the following example docker compose config file will contain this base config.
+
+## Example
+
+### Run [YCSB](https://github.com/pingcap/go-ycsb) to connect to the TiKV cluster:
```
Comment (overvenus) [inline docs/how-to/deploy/using-docker-compose.md]: Better to call it `GO-YCSB`.

**Comment by dcalvin** [comment]
PTAL again @overvenus

**Comment by dcalvin** [comment]
FYI @Hoverbear

File: docs/how-to/deploy/using-docker-compose.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
+---
+title: Install and Deploy TiKV Using Docker Compose
```
Comment (Hoverbear) [inline docs/how-to/deploy/using-docker-compose.md]: I think all these documents should probably end up on tikv.org instead, what do you think?

File: README.md [hunk 4]
```diff
@@ -71,22 +71,22 @@ We provide multiple deployment methods, but it is recommended to use our Ansible
...
 For the production environment, use [Ansible](https://github.com/pingcap/tidb-ansible) to deploy the cluster.
 
-- [Deploy TiDB Using Ansible](https://github.com/pingcap/docs/blob/master/op-guide/ansible-deployment.md)
-- [Deploy TiKV separately Using Ansible](docs/op-guide/deploy-tikv-using-ansible.md)
+- [Deploy TiDB Using Ansible](https://github.com/pingcap/docs/blob/master/dev/how-to/deploy/orchestrated/ansible.md)
+- [Deploy TiKV separately Using Ansible](docs/how-to/deploy/using-ansible.md)
 
 ## Client drivers
 
 Currently, the only interface to TiKV is the [TiDB Go client](https://github.com/pingcap/tidb/tree/master/store/tikv) and the [TiSpark Java client](https://github.com/pingcap/tispark/tree/master/tikv-client/src/main/java/com/pingcap/tikv).
```
Comment (Hoverbear) [inline README.md]:
```suggestion
There are the following clients for TiKV:

If you want to try the Go client, see [Try Two Types of APIs](docs/clients/go-client-api.md).
- [Go](https://github.com/tikv/client-go) (The most stable and widely used)
- [Java](https://github.com/tikv/client-java)
- [Rust](https://github.com/tikv/client-rust)
- [C](https://github.com/tikv/client-rust)

File: docs/how-to/deploy/using-docker-compose.md (hunk 3 above)
Comment (dcalvin) [reply docs/how-to/deploy/using-docker-compose.md]: Yeah. I agree.  I think we can finish updating the 3.0 topics for the release. And after that (very soon), we can start properly migrating it.

**Comment by dcalvin** [comment]
PTAL again @Hoverbear

**Pull request merged**